src/aind_codeocean_api/credentials.py

```diff
@@ -25,7 +31,63 @@ def _load_json(path: str) -> json:
 
     def __init__(self):
         """Initializes credentials."""
-        self.credentials_path = os.environ.get(
-            "CODEOCEAN_CREDENTIALS_PATH", "credentials.json"
-        )
-        self.credentials = self._load_json(self.credentials_path)
+
+        if os.environ.get(DEFAULT_ENV_VAR):
+            filepath = os.environ.get(DEFAULT_ENV_VAR)
+        else:
+            filepath = str(DEFAULT_HOME_PATH)
+
+        self.credentials = self._load_json(filepath)
+
+    @staticmethod
+    def create_credentials(
+        api_domain: str,
+        access_token: str,
+        file_location: str,
+    ) -> None:
+        """
+        Takes in credential information from
+        user input and create a credentials.json file.
+
+        Parameters
+        ---------------
+        api_domain : string
+            API domain
+        access_token : string
+            API Access Token
+        file_location : str
+            File path where credentials.json file is written to
+
+        Returns
+        ---------------
+        None
+            Writes to file
+        """
+
+        with open(file_location, "w+") as output:
+            json.dump(
+                {"domain": api_domain, "token": access_token}, output, indent=4
+            )
+
+
+if __name__ == "__main__":
+
+    # Prompt user
+
+    user_input_file_path = input("""Save to (leave blank to use default):""")
+    domain = input("Domain: ")
```

Comment:
can you add an example of what the domain looks like, e.g. the acmecorp one from your test? mostly to avoid confusion about whether to include the `https://` in the front and such.